packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -125,7 +125,7 @@ const defaultConfig = {
 	 * @kind member
 	 * @memberof ui/Marquee.MarqueeDecorator.defaultConfig
 	 */
-	marqueeDirection: (str) => direction(str) === 'rtl' ? 'rtl' : 'ltr'
+	marqueeDirection: (str) => isRtlText(str) ? 'rtl' : 'ltr'
```

Comment:
If we change this, is it necessary to keep https://github.com/enactjs/sandstone/blob/master/Marquee/Marquee.js#L25 ?

Reply:
OK